### PR TITLE
SAK-31123 - Web Service for copy content updated with Entity Transfer…

### PR DESCRIPTION
**Pull request by danielmerino**
…rer.

### CODE REVIEW

File: webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java [hunk 1]
```diff
@@ -4243,25 +4278,179 @@ public String copySiteContentForTool(
...
+    					}
+    					else
+    					{
+    						et.transferCopyEntities(fromContext, toContext,	new ArrayList(), true);
+    					}
+    				}
+    			}
+    			catch (Throwable t)
+    			{
+    				LOG.warn(this + ".transferCopyEntities: Error encountered while asking EntityTransfer to transferCopyEntities from: " + fromContext + " to: " + toContext, t);
```
Comment (ern) [inline webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java]: You can shorten the log messages as they will output the class and method so there is no need to include them in the log message.
`LOG.warn("Error encountered while asking EntityTransfer to transferCopyEntities from: " + fromContext + " to: " + toContext, t)`

**Comment by ern** [comment]
Other than the minor Log comment it looks good.

**Comment by danielmerino** [comment]
I have shortened log messages removing the class (I copied that log messages from SiteAction code).

I have updated too other obsolete log messages but I have left the method name because it was already there and I think that in my Tomcat logs I don't see method name, only class. Surely this can be configured but maybe others have the same misconfiguration and method name could be useful IMHO.

**Comment by ern** [comment]
You should see them if your running master.

**Pull request merged**